platform/features/listview/src/controllers/ListViewController.js

```diff
@@ -32,6 +32,10 @@ define(function () {
 
         this.utc = formatService.getFormat('utc');
 
+        setTimeout(function () {
+            $scope.$apply();
+        });
+
```

Comment:
@juliecwang - Seems like a good approach. Add comments to your code addition to make it easier for someone else to understand why we are explicitly calling $apply here.